packages/web-components/fast-components/src/disclosure/disclosure.styles.ts

```diff
@@ -0,0 +1,84 @@
+import { css } from "@microsoft/fast-element";
+import {
+    accentFillActiveBehavior,
+    accentFillHoverBehavior,
+    accentFillRestBehavior,
+    accentForegroundActiveBehavior,
+    accentForegroundCutRestBehavior,
+    accentForegroundHoverBehavior,
+    accentForegroundRestBehavior,
+} from "../styles/recipes";
+
+export const DisclosureStyles = css`
+    .disclosure {
+        transition: height 0.35s;
+    }
+
+    .disclosure .invoker::-webkit-details-marker {
+        display: none;
+    }
+
+    .disclosure .invoker {
+        list-style-type: none;
+    }
+
+    :host(.accent) .invoker {
+        background: ${accentFillRestBehavior.var};
+        color: ${accentForegroundCutRestBehavior.var};
+        font-family: var(--body-font);
+        font-size: var(--type-ramp-base-font-size);
+        border-radius: calc(var(--corner-radius) * 1px);
+        outline: none;
+        cursor: pointer;
+        margin: 16px 0;
+        padding: 12px;
+        max-width: max-content;
+    }
+
+    :host(.accent) .invoker:active {
```

Comment:
Can we move these host selectors to be attribute based instead of class based?

Reply:
Done :)